server/R/R_service.R

```diff
@@ -2,6 +2,8 @@
 
 library(jsonlite)
 library(httr)
+#library(textclean)
```

Comment:
Can we maybe delete this if we do not need it?

Reply:
Yep, done :) .  
Also removed the bit in R_service labelled "temporary hack to set url ..." as it was just a temporary hack..